### ISSUE TITLE
[BUG REPORT] - The AutoPause.exe does not terminate when the Reset button is clicked

### ISSUE THREAD

**Issue by amymor** [SegoCode/AutoWall#29]
### Description

The Reset button shuts down everything except autoPause.exe. I believe that including autoPause.exe in the Reset function would address the issue.

Off-topic, I have also modified autoPause to play video only when the desktop is active. As a result, when I select a window, the playback will pause. Here is the complete code if you're interested or would like to provide it as an additional version for users:
```
#NoTrayIcon

While 1
   $activeWindow = WinGetTitle("[ACTIVE]") ; Get the title of the active window
   If $activeWindow <> "" And $activeWindow <> "Desktop" Then ; If the active window is not the desktop
       ; Send the pause command to mpv
       Run(@ComSpec & " /c " & "echo set pause yes >\\.\pipe\mpvsocket", "", @SW_HIDE)
   Else
       ; Send the resume command to mpv
       Run(@ComSpec & " /c " & "echo set pause no >\\.\pipe\mpvsocket", "", @SW_HIDE)
   EndIf
   Sleep(1000) ; Check every second
WEnd
```

### Steps to reproduce

 1. Open app and set something as background.
 2. Press the Reset button.

### Windows version

Windows 10

### Confirmation

- [X] I performed a [search of the issue tracker](https://github.com/segocode/AutoWall/issues) to avoid opening a duplicate issue
- [X] I understand that not filling out this template correctly may lead to the issue being closed

**Comment by SegoCode** [comment]
Hey mate, 

How about if you open a PR and add that alternative version as "autoPause_ActiveWindows" or something like that, I'll compile it and add it to the readme, so if someone wants that functionality to replace the .exe

**Comment by amymor** [comment]
I'm not very familiar with GitHub, but I created a pull request. If there is any problem, please let me know.

**Comment by SegoCode** [comment]
> While I'm not very familiar with GitHub, I submitted a pull request. Please let me know if there are any problems.
No worrys, perfect, https://github.com/SegoCode/AutoWall/pull/30 merged, feel free to create any pr, thx!

**Comment by SegoCode** [comment]
Fixed on 1.12!
